#!/usr/bin/env python3

PACKAGE = "sema_gzsim"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("canny_threshold",   int_t,  0, "set canny_threshold",  45,     0,    100)
gen.add("blur_threshold",    int_t,  0, "set blur_threshold",    3,     0,    100)
gen.add("min_area",          int_t,  0, "set min_area",      10000, 10000,  50000)
gen.add("max_area",          int_t,  0, "set max_area",      50000, 10000,  50000)
gen.add("min_distance",      int_t,  0, "set min_distance",     15,     0,     50)


exit(gen.generate(PACKAGE, "dynamic_cfg", "box_detector"))